tests/datasets/test_cdc_events.py

```diff
@@ -130,11 +130,13 @@ def test_groups_join(
                         "btime": self.base_time,
                         "ntime": self.next_time,
                     },
+                    "tenant_ids": {"organization_id": 1, "referrer": "abcd"},
```

Comment:
Not specific to your PR. But at some point of time we should probably just get rid of all CDC related code since it is unused.